src/common/stattextcmn.cpp

```diff
@@ -145,7 +147,13 @@ void wxTextWrapper::Wrap(wxWindow *win, const wxString& text, int widthMax)
             }
 
             // Find the last word to chop off.
-            const size_t lastSpace = line.rfind(' ', posEnd);
+            const size_t lastSpace1 = line.rfind(' ', posEnd);
+            const size_t lastSpace2 = line.rfind(zeroWidthSpace, posEnd);
```

Comment:
There are many more than two whitespace characters out there, some of them are wrappable but not all: https://en.wikipedia.org/wiki/Whitespace_character also internalization depends on the locale... the proper way to find breaking opportunities would be to use ICU's BreakIterator: http://icu-project.org/apiref/icu4c/classicu_1_1BreakIterator.html#adffa57ea9131c4a9e0cf8226c17d459a

Here's a random snippet that does word wrapping with ICU http://www.java2s.com/Tutorial/Java/0220__I18N/WordWrap.htm

Reply:
Btw line breaking is specified by [UAX #14](http://unicode.org/reports/tr14/).